programmers/모의고사.js

```diff
@@ -0,0 +1,31 @@
+function solution(answers) {
+    
+    const firstAns = [1,2,3,4,5]; 
+    const secondAns = [2,1,2,3,2,4,2,5]; 
+    const thirdAns = [3,3,1,1,2,2,4,4,5,5]; 
+    
+    let first = 0;
+    let second = 0;
+    let third = 0;
```

Comment:
```suggestion
    const score = [0, 0, 0];
    // or
    let [first, second, third] = [0, 0, 0];
```
이렇게 사용해서 1,2,3번째를 표현할 수도 있어요
+1할때는 score[0] += 1 이렇게 쓰면 됩니다.